composer.json

```diff
@@ -17,7 +17,8 @@
     "require": {
         "php": ">=5.6.0",
         "jasny/php-functions": "^2.0",
-        "psr/http-message": "^1.0"
+        "psr/http-message": "^1.0",
+        "phpunit/phpunit": "^5.5"
```

Comment:
phpunit should be in `requireDev`. However please use `jasny/php-code-quality` instead. That package has phpunit as dependency.